tests/test_alchemy.py

```diff
@@ -13,6 +13,32 @@
 from .alchemyapp import models
 
 
+def setUpModule():
```

Comment:
Full disclosure: I know very little of sqlalchemy, I ran into a lot of issues, this is the only way I could get it working reliably, **feedback on this section is a must!!**